src/pages/Account/ProfileQuestions.tsx

```diff
@@ -34,33 +32,29 @@ export interface QuestionsFormData {
 
 export const ProfileQuestions: React.FC = () => {
   const history = useHistory();
-  const location = useLocation();
 
   const { user } = useUser();
 
-  const { spaceSlug } = useSpaceParams();
-  const { space } = useSpaceBySlug(spaceSlug);
-
-  const { world, isLoaded: isWorldLoaded } = useCurrentWorld({
-    worldId: space?.worldId,
-  });
+  const { worldSlug, spaceSlug } = useSpaceParams();
+  const { world, space, isLoaded } = useSpaceBySlug(worldSlug, spaceSlug);
 
   const { register, handleSubmit, formState } = useForm<QuestionsFormData>({
     mode: "onChange",
   });
 
   const proceed = useCallback(() => {
-    history.push(`${ACCOUNT_CODE_QUESTIONS_URL}${location.search}`);
-  }, [history, location.search]);
+    const nextUrl = accountCodeQuestionsUrl(worldSlug, spaceSlug);
+    history.push(nextUrl);
```

Comment:
variable can be inlined, block dropped...

Reply:
Resolved in 124ca0ef